Lombiq.UIKit.Widgets/Recipes/Lombiq.UIKit.Widgets.Content.recipe.json

```diff
@@ -0,0 +1,191 @@
+{
+  "name": "Lombiq.UIKit.Widgets.Content",
+  "displayName": "Lombiq UI Kit widgets - Sample Content - Carousel Widget",
+  "description": "Sample content for testing and demonstration of the Lombiq UI kit Widgets module.",
+  "author": "Lombiq Technologies",
+  "website": "https://github.com/Lombiq/Orchard-UIKit",
+  "version": "1.0",
+  "issetuprecipe": false,
+  "categories": [
+    "training"
+  ],
+  "tags": [
+    "developer",
+    "training",
+    "sample"
+  ],
+  "steps": [
+    {
+      "name": "feature",
+      "enable": [
+        "Lombiq.UIKit.Widgets"
```

Comment:
Also enable the Helpful Content Types feature because the Page content type won't be loaded otherwise.